pythonExecSource/src/main/python/pyExecConnector.py

```diff
@@ -673,6 +681,11 @@ async def run(self):
             pec.establish_handlers()
 
         self.logger.info('Running PyExecConnector.')
```

Comment:
Look for env variable set up by K8s to determine whether to start the TCP probe handler